lib/vanagon/platform/windows.rb

```diff
@@ -46,6 +65,20 @@ def generate_packaging_artifacts(workdir, name, binding)
         end
       end
 
+      # Method to generate the files required to build a windows msi package for the project
+      #
+      # @param workdir [String] working directory to stage the evaluated templates in
+      # @param name [String] name of the project
+      # @param binding [Binding] binding to use in evaluating the packaging templates
+      def generate_packaging_artifacts_msi(workdir, name, binding)
+        # nuget templates that do require a name change
+        erb_file(File.join(VANAGON_ROOT, "resources/windows/msi/service-filter.xslt.erb"), File.join(workdir, "#{name}.service-filter.xslt"), false, { :binding => binding })
+        erb_file(File.join(VANAGON_ROOT, "resources/windows/msi/main.wxs.erb"), File.join(workdir, "#{name}.main.wxs"), false, { :binding => binding })
```

Comment:
I would change the order here and put main.wxs.erb first, as its the most important one and it has to work. As this list gets longer (feature, directory, customaction, misc components, etc) to have it mixed in confuses things.